deployment/aws-content-localization-web.yaml

```diff
@@ -130,7 +130,7 @@ Resources:
             Resource: !Sub "arn:aws:s3:::${ContentAnalysisWebsiteBucket}/*"
             Condition:
               Bool:
-                aws:SecureTransport: false
+                aws:SecureTransport: true
```

Comment:
I'm looking at this support page: https://aws.amazon.com/premiumsupport/knowledge-center/s3-bucket-policy-for-config-rule/

This rule reads: deny if secure transport is true
Should it be: Deny if secure transport is false? (i.e. no change needed)

Also,  should the bucket be included in the policy statement as well?  ""arn:aws:s3:::${ContentAnalysisWebsiteBucket}"